src/controller/python/chip-device-ctrl.py

```diff
@@ -358,6 +362,41 @@ def do_btpconnect(self, line):
             return
 
         print("BTP Connected")
+    
+    def do_connect(self, line):
+        """
+        connect (rendezvous)
+        """
+
+        try:
+            args = shlex.split(line)
+            if len(args) == 0:
```

Comment:
does `if not args:` work?

Reply:
I prefer len(args) since it is clearer than `if not args`

Reply:
It is more pythonic to not have len: https://google.github.io/styleguide/pyguide.html#214-truefalse-evaluations